### PR TITLE
Fixing README.md file

### PR DESCRIPTION
**Pull request by GJZ26**
Is this the introductory documentation for Next.js or Instagram Clone??? smh

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,27 +27,39 @@ pnpm dev
...
 - Encontrarás un apartado de `components/`los cuales son todos aquellos componentes que se introducirán al archivo `pages/index.js` para posteriormente renderizar la página.
-> **Note**
 
+> **Note**
 > - La [documentación de Next.js](https://nextjs.org/docs) nos recomienda estar familiarizado con `Javascript` y `React` anteriormente.
 
 
 # Previsualización:
 
-- `Feed`:
```
Comment (GJZ26) [inline README.md:45]: Aunque a veces es importante enlistar cosas, debes respetar la semántica de Markdown para facilitar la lectura

Comment (Leonardo-Toledo-V) [reply README.md:45]: pinche mamón

**Review by Leonardo-Toledo-V — CHANGES_REQUESTED** [review]
Bro, I don't need anyone to tell me what to do in my repo, please don't talk to me.

**Comment by Leonardo-Toledo-V** [comment]
Shut up bitch

**Comment by GJZ26** [comment]
> Bro, I don't need anyone to tell me what to do in my repo, please don't talk to me.

Accept my pull request, you bastard, mamahuevo

File: README.md (hunk 1 above)
Comment (GJZ26) [reply README.md:45]: cálleseeee mamahuevoooo

**Pull request merged**